core/src/main/java/org/ocast/core/PrivateSettings.java

```diff
@@ -60,7 +64,11 @@
 
     void reset(Consumer<Integer> onSuccess, Consumer<Throwable> onFailure);
 
-    void checkStick(Consumer<Integer> onSuccess, Consumer<Throwable> onFailure);
+    void checkStick(int op, Consumer<CheckStickReply> onSuccess, Consumer<Throwable> onFailure);
```

Comment:
Method signature has changed. How can we manage backward compatibility ? (not even sure it is possible)